src/components/v5/common/AvatarUploader/partials/ProgressContent.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import clsx from 'clsx';
+
+import Icon from '~shared/Icon';
+import styles from './AvatarUploader.module.css';
+import { ProgressContentProps } from '../types';
+import ProgressBar from '~v5/shared/ProgressBar';
+
+const displayName = 'v5.common.AvatarUploader.partials.ProgressContent';
+
+const ProgressContent: FC<ProgressContentProps> = ({
+  progress,
+  handleFileRemove,
+  fileName,
+  fileSize,
+}) => {
+  const isUploadCompleted = progress === 100;
```

Comment:
Not sure if this matter of style is used across this project but maybe for the clarity we could stick to adding an empty line above return statements